app/helpers/eq.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+import { equalHelper } from 'ember-freestyle/helpers/equal';
+
+let forExport = null;
+
+if (Ember.Helper) {
+  forExport = Ember.Helper.helper(equalHelper);
+} else if (Ember.HTMLBars.makeBoundHelper) {
+  forExport = Ember.HTMLBars.makeBoundHelper(equalHelper);
+}
+
+export default forExport;
```

Comment:
This implementation will conflict with an implementation from ember-truth-helpers if an app uses both freestyle and ember-truth-helpers. Consider making the helper `freestyle-eq` OR investigate whether there is a way for a helper to be included in addons/ tree only.